examples/plot_random_vs_gp.py

```diff
@@ -0,0 +1,55 @@
+"""
+==========================================
+Tying a sklearn estimator with gp_minimize
+==========================================
+
+This example shows how to chain gp_minimize with RandomForestClassifier
+in this case.
+"""
+print(__doc__)
+from time import time
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.stats import randint as sp_randint
+
+from sklearn.base import clone
+from sklearn.datasets import load_digits
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import GridSearchCV
+from sklearn.model_selection import RandomizedSearchCV
+
+from skopt.gp_opt import gp_minimize
+
+digits = load_digits()
+X, y = digits.data, digits.target
+rfc = RandomForestClassifier(random_state=10)
+
+def compute_mean_validation_score(forest_params):
+    # Hack to allow integer parameters, since the parameters
+    # sampled internally are uniform in a given range.
```

Comment:
This would worth of an issue, to discuss about the best API to define parameter types.